tests/cases/fourslash/server/autoImportProvider_pnpm.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="../fourslash.ts" />
+
```

Comment:
Definitely add test with preserveSymLink set to true ?

Reply:
pnpm doesn’t work with `--preserveSymlinks`. package.json auto imports probably does need more testing with `--preserveSymlinks` in general, but I want it to be an actually realistic test.